src/Umbraco.Web.UI/config/umbracoSettings.config

```diff
@@ -104,7 +104,8 @@
 
     <!-- Defines the default document type property used when adding properties in the back-office (if missing or empty, defaults to Textstring -->
     <defaultDocumentTypeProperty>Textstring</defaultDocumentTypeProperty>
-    
+
+    <showDeprecatedPropertyEditors>true</showDeprecatedPropertyEditors>
```

Comment:
Not a mistake, this is the debug version of the config, the release doesn't contain this and it is false.